mapper/orm-jakarta-batch/core/src/main/java/org/hibernate/search/jakarta/batch/core/massindexing/util/impl/PersistenceUtil.java

```diff
@@ -70,6 +70,7 @@ public static StatelessSession openStatelessSession(EntityManagerFactory entityM
 		@SuppressWarnings("rawtypes")
 		StatelessSessionBuilder builder = sessionFactory.withStatelessOptions();
 		if ( StringHelper.isNotEmpty( tenantId ) ) {
+			// TODO: needs a converter here too!
 			builder.tenantIdentifier( (Object) tenantId );
```

Comment:
I'm not sure if we could pass the converter to these batch utils...  or say something like "the converter should be a named bean "blablabla" and try injecting it as:
```java
@Inject
@Named("blablabla")
private TenantIdentifierConverter tenantIdentifierConverter;
```